modules/apps/frontend-theme/frontend-theme-contributor-extender/src/main/java/com/liferay/frontend/theme/contributor/extender/internal/servlet/taglib/ThemeContributorTopHeadDynamicInclude.java

```diff
@@ -69,7 +71,11 @@ public void include(
 			portalCDNURL = themeDisplay.getPortalURL();
 		}
 
-		if (_cssResourceURLs.length > 0) {
+		ResourceURLsBag resourceURLsBag = _getResourceURLsBag();
+
+		List<String> cssResourceURLs = resourceURLsBag._cssResourceURLs;
+
+		if (!cssResourceURLs.isEmpty()) {
```

Comment:
@W2jx The only change compared to the last merged one is this line, right?

Reply:
Yes